docs/UserGuide.md

```diff
@@ -263,6 +263,25 @@ To create a deadline named `event proposal` with due time at today's 6 pm.
 
 To create a deadline named `EE2024 homework 1` at 2016 1st October 6 am.
 
+### List tasks: `list`
+
+List certain type of task stored in the database
```

Comment:
Since our UI already has three panes for displaying events, deadlines and floating tasks, why not just have `list` list all of them at once?